sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -3419,6 +3419,39 @@ class Dataset[T] private[sql](
     } (Encoders.STRING)
   }
 
+  /**
+   * Returns the content of the Dataset as a Dataset of JSON strings.
+   * @since 3.0.1
+   */
+  def toJSON(options: Map[String, String]): Dataset[String] = {
```

Comment:
I would deduplicate it by calling `toJSON(options: Map[String, String])` at `toJSON`.

Reply:
ok!